docker-compose.yml

```
@@ -36,7 +36,7 @@ services:
     restart: on-failure
   postgres:
     container_name: lenra-postgres
-    image: postgres:13
+    image: postgres:12
```

Why did you downgraded the Postgres version ? We use Postgres 13 every where (even in production):
```suggestion
    image: postgres:13
```

---

nvm it's working.